builtin/providers/postgresql/resource_postgresql_db.go

```diff
@@ -0,0 +1,27 @@
+package postgresql
+
+import (
+	"github.com/hashicorp/terraform/helper/schema"
+)
+
+func resourcePostgresqlDb() *schema.Resource {
+	return &schema.Resource{
+		Create: resourcePostgresqlDbCreate,
+		Read:   resourcePostgresqlDbRead,
+		Update: resourcePostgresqlDbUpdate,
+		Delete: resourcePostgresqlDbDelete,
+
+		Schema: map[string]*schema.Schema{
+			"name": &schema.Schema{
+				Type:     schema.TypeString,
+				Required: true,
+				ForceNew: true,
+			},
+			"owner": &schema.Schema{
+				Type:     schema.TypeString,
+				Required: true,
+				ForceNew: false,
+			},
```

Comment:
I don't know Postgres too well so please forgive me if this is a strange question: in Terraform resources we usually try to keep the number of required fields down to a minimum so that the "simple case" of configuration is as small as possible. Is it acceptable to leave `owner` unset and have Postgres e.g. assume ownership by the user that was specified in the provider configuration?

On the other hand, if setting the owner on a database is something a Postgres user would expect to alwayss have to do then I think that's fine. :smile: